src/server/pps/server/master.go

```diff
@@ -238,6 +238,10 @@ func (a *apiServer) master() {
 			}
 		}
 	}, backoff.NewInfiniteBackOff(), func(err error, d time.Duration) error {
+		for _, c := range a.monitorCancels {
+			c()
+		}
+		a.monitorCancels = make(map[string]func())
```

Comment:
This PR might just not be ready yet, but does this map get added to anywhere?

Reply:
It does, you're not going to see it in the diff though because the code was already there and hasn't changed. The main change here is that we're cancelling all the monitor goros, and clearing the map when the master process goes into backoff. This is because during this time we're not master so we shouldn't be doing master things such as making `cron` commits.